Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -607,10 +568,35 @@ function (FixerInterface $a, FixerInterface $b) {
             }
         }
 
+        $rulesSets = array();
+        foreach (RulesSet::create()->getSetDefinitionNames() as $setName) {
```

Comment:
Accepted ;)

But it looks wrong to me to have a method for getting all ruleset names in the interface for _one_ ruleset (`RulesSetInterface`). And IMO it can hurt us later, too.